docs/database-restore.md

```diff
@@ -0,0 +1,49 @@
+# Apply for Postgraduate Teacher Training - Manual Deployment
+
+## Purpose
+
+This document describes the process of restoring the database from the Azure backups.
+
+Azure retains Postgres database backups for up to 35 days depending upon the configuration used. The Apply production environment has the full 35 days backup, the other environments have the retention period reduced to seven days.
+
+This document assumes that only the data in the database itself is at fault and the database server still exists.
+
+## Prerequisites
+
+To carry out a production database restore routine you will be required to have the following software installed:
+- PostgreSQL 9.6.x client (specifically `pg_dump` and `pg_restore`)
+
+## Instructions
+
+**NOTE: Before following the steps below you will need to request an elevation of your rights to the 'contributor' role through PIM in the Azure Portal if working on an app hosted in the test or production subscriptions. Guidance on PIM can be found at the end of this document. PIM is not required in the development subscription.**
+
+These instructions assume that you are working on the production environment, but they apply equally to the other environments by substitution of the resource names prefix for the chosen environment.
+
+1. Launch the [Azure Portal](https://portal.azure.com).
+1. Go to the "Resource groups" blade.
+1. Select the `s106p01-apply` resource group.
+1. Select the Azure Database for PostgreSQL server database named `s106p01-apply-psql` from the list of resources.
+1. Click the "Restore" button in the ribon at the top of the database resource page.
+1. Select the point in time you wish to restore to and enter a name for the new database server, e.g. `s106p01-apply-psql-restored`. The naming prefix convention must be adhered to; CIP's policy enforcement will prevent the use of non-conforming names.
+1. Click the OK button to start the restore process. This will take in the region of ten to fifteen minutes to completely retore the database to a new server, depending upon the size of the database.
```

Comment:
```suggestion
1. Click the OK button to start the restore process. This will take in the region of ten to fifteen minutes to completely restore the database to a new server, depending upon the size of the database.
```